Google.PowerShell/BigQuery/BqTable.cs

```diff
@@ -163,6 +163,63 @@ public void DoGetRequest(TablesResource.GetRequest request)
 
     /// <summary>
     /// <para type="synopsis">
+    /// Updates information describing an existing BigQuery table.
+    /// </para>
+    /// <para type="description">
+    /// text
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> $my_tab = Get-BqTable -DatasetId “my_data” “my_table”
```

Comment:
For multi-line examples like this, give the start and end tags their own lines, and indent properly. `<code>` preserves whitespace, except it also trims down to relevant text.  Just keep consistent indentation, and don't worry about empty starting and trailing newlines. Follow normal rules for how much to indent, and have nice alignment within that indent.